system-parachains/bridge-hubs/bridge-hub-polkadot/src/lib.rs

```diff
@@ -122,7 +122,8 @@ pub type SignedExtra = (
 	frame_system::CheckWeight<Runtime>,
 	pallet_transaction_payment::ChargeTransactionPayment<Runtime>,
 	BridgeRejectObsoleteHeadersAndMessages,
-	bridge_to_kusama_config::RefundBridgeHubKusamaMessages,
+	bridge_to_kusama_config::OnBridgeHubPolkadotRefundBridgeHubKusamaMessages,
+	cumulus_primitives_storage_weight_reclaim::StorageWeightReclaim<Runtime>,
```

Comment:
@pandres95 why are you adding `StorageWeightReclaim` as a part of the bump? I mean, there is a separate [issue](https://github.com/polkadot-fellows/runtimes/issues/494) for that, so I am suggesting to revert it from this PR.

Reply:
and also I think it should be added after `CheckMetadataHash` at the end

Reply:
Reverting